<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page_main_window">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>TEMP ADMIN LOGIN:

username: admin
password: password</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>288</width>
        <height>198</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="viewListWineriesButton">
         <property name="text">
          <string>View Our List of Wineries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="planDayTripButton">
         <property name="text">
          <string>Plan A Day Trip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tourWineriesButton">
         <property name="text">
          <string>Tour Your Wineries and Purchase Wines</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="adminLogButton">
         <property name="text">
          <string>Admin Log-In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>631</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Comic Sans MS'; font-size:18pt;&quot;&gt;STRAIGHT AHEAD WINE TOURS OF THE CENTRAL COAST&lt;/span&gt;&lt;span style=&quot; font-size:18pt;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_view_list_wineries">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>View Wineries Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ViewBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_plan_day_trip">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Plan Day Trip Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PlanTripBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_tour_and_purchase">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tour and Purchase Wine Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TourBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_admin_login">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>521</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_admin_login_form">
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>210</y>
          <width>209</width>
          <height>58</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>User Name :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userNameLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Password   :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordLine">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>150</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Admin Log In</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>290</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="admin_log_ok_button">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="adming_log_cancel_button">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_admin_login_success">
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>190</y>
          <width>261</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Congrats, you are now logged in</string>
        </property>
       </widget>
       <widget class="QPushButton" name="admin_log_out_button">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>270</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
       <widget class="QPushButton" name="LogInBack">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>230</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Back to Main</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7"/>
    <widget class="QWidget" name="page_8"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
